src/lib/containers/widgets/Range.tsx

```diff
@@ -37,13 +40,18 @@ export const Range: React.FunctionComponent<RangeProps> = (
 
     type: ControlType = 'number',
   ) => {
+    if (!values.min || !values.max) {
```

Comment:
I think this needs to explicitly check whether min or max is undefined, otherwise if min or max is zero this will incorrectly show an error message